src/main/java/edu/utdallas/davisbase/storage/IndexPage.java

```diff
@@ -0,0 +1,497 @@
+package edu.utdallas.davisbase.storage;
+
+import java.io.IOException;
+import java.io.RandomAccessFile;
+import java.util.ArrayList;
+import java.util.List;
+
+public class IndexPage {
+
+	static final int BYTES_OF_PAGE_OFFSET = Short.BYTES;
+	static final int PAGE_OFFSET_OF_CELL_COUNT = 0x01;
+	static final int PAGE_OFFSET_OF_RIGHTMOST_PAGE_NO = 0x06;
+	static final int PAGE_OFFSET_OF_CELL_PAGE_OFFSET_ARRAY = 0x10;
+	static final int PAGE_OFFSET_OF_CELL_CONTENT_START_POINT = 0x03;
+	static final int PAGE_OFFSET_OF_CELL_PARENT_PAGE_NO = 0x0A;
+	static final int pageSize = StorageConfiguration.Builder.getDefaultPageSize();
```

Comment:
Make `pageSize` an injected dependency. Do *not* set it yourself using an _ad hoc_ reference to the default.